app/policies/ppiu_policy.rb

```diff
@@ -4,7 +4,7 @@
 
 PPIUPolicy = Struct.new(:user, :ppiu) do
   def access?
-    user.loa3? && user.multifactor && Flipper.enabled?(:direct_deposit_cnp, user)
+    user.loa3? && user.identity.sign_in[:service_name] == 'idme' && Flipper.enabled?(:direct_deposit_cnp, user)
```

Comment:
Mobile users don't go through the same provider as .gov users. This means that users who select id.me as the sign in method will have 'oauth_IDME' returned for `user.identity.sign_in[:service_name]`.
```suggestion
    user.loa3? && ['idme', 'oauth_IDME'].include?(user.identity.sign_in[:service_name]) && Flipper.enabled?(:direct_deposit_cnp, user)
```